interoptest/src/testcoordinator/validator/trace_constructor.go

```diff
@@ -149,7 +150,7 @@ func spanToSimpleSpan(span *tracepb.Span) *SimpleSpan {
 }
 
 func isRoot(span *tracepb.Span) bool {
-	return span.ParentSpanId == nil || len(span.ParentSpanId) == 0
+	return span.ParentSpanId == nil || len(span.ParentSpanId) == 0 || zeroSpan == ToSpanID(span.ParentSpanId)
```

Comment:
parent span id is zero but will the span id ever be zero? That would be invalid span. It should be thrown away during groupSpansByTraceID. what do you think?

Reply:
In Java if no implementation is provided, all the spans will have an ID of 0: https://github.com/census-instrumentation/opencensus-java/blob/master/api/src/main/java/io/opencensus/trace/BlankSpan.java#L41. This is an edge case - it's hard to tell if that is invalid or not, because when integrating OC-Java with other Java libraries, usually no implementation is provided.

Anyway from the perspective of interop tests I think we should always test against a real implementation, so I'm fine with leaving this case as invalid for now.